src/components/forms/Switch/Switch.stories.tsx

```diff
@@ -1,32 +1,145 @@
-import React from 'react';
 import { Meta, Story } from '@storybook/react/types-6-0';
+import React, { useState } from 'react';
 
+import { generateControl } from '../../../utils/tests/storybook';
+import { Stack } from '../../layout/Stack';
+import { Text } from '../../typography';
 import Switch from './Switch';
+import { SwitchSizes } from './Switch.enums';
 import { SwitchProps } from './Switch.types';
 
 export default {
   title: 'components/forms/Switch',
   component: Switch,
+  argTypes: {
+    maxWidth: { control: { type: 'number' } },
+    defaultChecked: { control: { type: 'boolean' } },
+    size: {
+      ...generateControl('select', SwitchSizes),
+    },
+  },
 } as Meta;
 
 export const playground: Story<SwitchProps> = (args) => <Switch {...args} />;
 playground.parameters = {
   chromatic: { disable: true },
 };
-playground.args = { label: 'Public', switchId: 'playground' };
+playground.args = {
+  label: 'Public',
+  switchId: 'playground',
+};
 playground.argTypes = {
   label: {
     description: 'Text in switch',
   },
   switchId: {
     description: 'ID to connect input and its label',
   },
+  maxWidth: {
+    description: 'Max width in px',
+  },
 };
 
-export const Default: Story = () => (
-  <Switch label="Private" switchId="default" />
+export const Default: Story = () => <Switch switchId="default" />;
```

Comment:
I miss a story with sizes showcase.

Reply:
I added a small showcase for each Story 👍